foodnet/membership/tests.py

```diff
@@ -35,15 +47,15 @@ def setUp(self):
         email_address.save()
 
     def test_profile(self):
-        resp = self.client.get(reverse('profile'))
+        response = self.client.get(reverse('profile'))
```

Comment:
I really disagree :) and this is why I want us to have a style guide so we're all on the same page. The reason I rename `resp` to `response` is because it makes the code clearer. Code is read more than written, so having meaningful names for variables is in my opinion very important for future maintenance.